core/domain/exp_domain.py

```diff
@@ -1401,7 +1406,7 @@ def __init__(
         self.created_on = created_on
         self.last_updated = last_updated
         self.auto_tts_enabled = auto_tts_enabled
-        self.correctness_feedback_enabled = correctness_feedback_enabled
+        self.correctness_feedback_enabled = True
```

Comment:
Why does correctness_feedback_enabled still need to be a property of the Exploration domain object? We should just drop it, right? (We didn't persist other deprecated fields at this layer.)

Reply:
I believe other properties, like next_content_id_index still persist in the domain object. that's why I have added it.

Reply:
next_content_id_index is still in use; this one is not. Please drop.

Reply:
Done.